report.php

```diff
@@ -8,279 +8,236 @@
  *
  */
 
-define("IN_MYBB", 1);
+define('IN_MYBB', 1);
 define('THIS_SCRIPT', 'report.php');
 
-$templatelist = "report,report_thanks,report_error,report_reasons,report_error_nomodal,forumdisplay_password_wrongpass,forumdisplay_password";
-require_once "./global.php";
-require_once MYBB_ROOT.'inc/functions_modcp.php';
+$templatelist = 'report,report_thanks,report_error,report_reasons,report_error_nomodal,forumdisplay_password_wrongpass,forumdisplay_password';
```

Comment:
This is unnecessary right now, right?

Reply:
Yes. I just got an answer on Discord from Euan. I will handle it tonight.

Reply:
Done @Matslom